WcaOnRails/app/controllers/registrations_controller.rb

```diff
@@ -534,7 +534,14 @@ def stripe_webhook
     else
       logger.info "Unhandled Stripe event type: #{event.type}"
     end
-
+    if stored_intent.holder.type == "attendee"
```

Comment:
This won't work. Calling `holder` immediately prompts ActiveRecord to try and parse the columns `holder_id` and `holder_type` into a model called `attendee` which doesn't exist. An error would be thrown even before checking `type` is possible.